features/hbx_admin/manage_sep_types/sep_type_visibility.feature

```diff
@@ -1,77 +1,34 @@
 Feature: Admin has ability to create a new SEP Type with visibility options for "Customer & Admin" and "Admin Only"
   Background:
     Given both shop and fehb market configurations are enabled
-    Given all market kinds are enabled for user to select
-    Given all announcements are enabled for user to select
-    Given that a user with a HBX staff role with hbx_tier3 subrole exists
-    When Hbx Admin logs on to the Hbx Portal
-    Given the Admin is on the Main Page
+    And all market kinds are enabled for user to select
+    And all announcements are enabled for user to select
+    And that a user with a HBX staff role with hbx_tier3 subrole exists
+    And Hbx Admin logs on to the Hbx Portal
+    And the Admin is on the Main Page
     And the FAA feature configuration is enabled
     And Qualifying life events of all markets are present
     And the Admin will see the Manage SEPs under admin dropdown
     And Admin can click Manage SEPs link
```

Comment:
These 3 steps seem unnecessary.

Reply:
even though they seem like we are just checking, we are creating seps and clicking manage sep link dropdown